src/neovim/neovim.ts

```diff
@@ -59,25 +63,31 @@ export class Neovim implements vscode.Disposable {
     if (!(await util.promisify(exists)(dir))) {
       dir = __dirname;
     }
-    this.process = spawn(configuration.neovimPath, ['-u', 'NONE', '-i', 'NONE', '-N', '--embed'], {
+    this.process = spawn(configuration.neovimPath, ['-u', 'NONE', '-n', '--embed', '--headless'], {
```

Comment:
so now that I am using --headless do I still need the -i for SHADA stuff? A bit confused here

Reply:
`-i NONE` is not needed for 0.3.2.

In 0.3.1,  `--headless` is useless; in 0.3.2+ it is worse than useless because it changes the behavior of `--embed`.

Reply:
Got it, there is not a great way to spawn the process, then check the version to know whether or not -i NONE is needed, I am ok with just leaving it omitted for now (since we also got away without too many bug reports for many many months) if it only will impact 0.3.1 users